components/os/unix.go

```diff
@@ -22,6 +23,10 @@ func (u *Unix) GetDefaultInstanceType(arch Architecture) string {
 }
 func (*Unix) GetAgentConfigFolder() string { return "/etc/datadog-agent" }
 
+func (*Unix) CheckIsAbsPath(path string) bool {
```

Comment:
I don't think it works if you create a Ubuntu VM from a Windows laptop as `filepath.IsAbs` would return if the path is a Windows absolute path.

Reply:
👍 

Fixed in https://github.com/DataDog/test-infra-definitions/pull/289/commits/62f077c5fa464c54e484f2f229435941058b6158